src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -759,6 +759,7 @@ Table: Program indicator components
 | Analytics type | The available analytics types are *event* and *enrollment*.<br> <br>The analytics type defines whether the program indicator is calculated based on events or program enrollments. This has an impact on what type of calculations can be made.<br> * Events implies a data source where each event exists as an independent row. This is suitable for performing aggregations such as counts and sums.<br> * Enrollments implies a data source where all events for a single enrollment is combined on the same row. This allows for calculations which can compare event data from various program stages within a program enrollment. |
+| Organisation unit field | Determines which organisation unit is assigned to program indicator values.<br><br> For Event programs (without registration) the options are:<br> * Event organisation unit (default): where the event took place<br> * any data elements of value type Organisation Unit (if any) assigned to the program<br><br> For Tracker programs (with registration) and analytics type *Enrollment* the options are:<br> * Enrollment organisation unit (default): where the tracked entitiy instance was enrolled in this program<br> * Registration organisation unit: where the tracked entity instance was created<br> * Owner at start organisation unit: where the tracked entity instance was owned at the start of the reporting period<br> * Owner at end organisation unit: where the tracked entity instance was owned at the end of the reporting period<br><br> For Tracker programs (with registration) and analytics type *Event* the options are:<br> * Event organisation unit (default): where the event took place<br>  * any data elements of value type Organisation Unit (if any) assigned to the program<br> * Registration organisation unit: where the tracked entity instance was created<br> * Enrollment organisation unit: where the tracked entitiy instance was enrolled in this program<br> * Owner at start organisation unit: where the tracked entity instance was owned at the start of the reporting period<br> * Owner at end organisation unit: where the tracked entity instance was owned at the end of the reporting period |
```

Comment:
re-order so that the duplicated options in both tracker programs are in the same order. Plus a couple of typos
```suggestion
| Organisation unit field | Determines which organisation unit is assigned to program indicator values.<br><br> For Event programs (without registration) the options are:<br> * Event organisation unit (default): where the event took place<br> * any data elements of value type Organisation Unit (if any) assigned to the program<br><br> For Tracker programs (with registration) and analytics type *Enrollment* the options are:<br> * Registration organisation unit: where the tracked entity instance was created<br> * Enrollment organisation unit (default): where the tracked entity instance was enrolled in this program<br> * Owner at start organisation unit: where the tracked entity instance was owned at the start of the reporting period<br> * Owner at end organisation unit: where the tracked entity instance was owned at the end of the reporting period<br><br> For Tracker programs (with registration) and analytics type *Event* the options are:<br> * Event organisation unit (default): where the event took place<br>  * any data elements of value type Organisation Unit (if any) assigned to the program<br> * Registration organisation unit: where the tracked entity instance was created<br> * Enrollment organisation unit: where the tracked entity instance was enrolled in this program<br> * Owner at start organisation unit: where the tracked entity instance was owned at the start of the reporting period<br> * Owner at end organisation unit: where the tracked entity instance was owned at the end of the reporting period |
```